rocky/katalogus/client.py

```diff
@@ -84,6 +84,7 @@ def __init__(self, base_uri: str, organization: str):
         self.session = httpx.Client(base_url=base_uri)
         self.organization = organization
         self.organization_uri = f"/v1/organisations/{organization}"
+        self.logger = logger.bind(organization_code=organization)
```

Comment:
I think it would better to add the organization code to the context in OrganizationView once instead of having to add it to a lot of different places. We can also use the module level logger here.

Reply:
Should be fixed now